src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,25 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozerList = new ArrayList<>();
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
```suggestion
    public List<Buldozer> get() {
```
concrete produces produces concrete machines. fox in all places